template.yaml

```diff
@@ -622,6 +626,27 @@ Resources:
 
   #==========================Event Lambda functions====================================================
 
+  BrageMigrationEventConsumer:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: brage-import
+      Handler: no.sikt.nva.brage.migration.lambda.BrageEntryEventConsumer::handleRequest
+      Runtime: java11
+      ReservedConcurrentExecutions: 1
+      Role: !GetAtt LambdaRole.Arn
+      Events:
+        BrageMigrationEvent:
+          Type: S3
+          Properties:
+            Bucket: !Ref BrageMigrationInputBucket
+            Events: 's3:ObjectCreated:*'
```

Comment:
Maybe we only want new objects? Not sure what the intention is

Reply:
fixed